server/lib/payments.ts

```diff
@@ -207,24 +214,27 @@ export const buildRefundForTransaction = (
   refund.CreatedByUserId = user?.id || null;
   refund.description = `Refund of "${t.description}"`;
   refund.data = { ...refund.data, ...data };
+  refund.isRefund = true;
+  refund.hostFeeInHostCurrency = 0; // We're handling host fees in separate transactions
 
   /* The refund operation moves back fees to the user's ledger so the
    * fees there should be positive. Since they're usually in negative,
    * we're just setting them to positive by adding a - sign in front
    * of it. */
-  refund.hostFeeInHostCurrency = -refund.hostFeeInHostCurrency;
```

Comment:
I think this is here to guarantee backward compatibility because `refundHostFee()` only works with newer transactions with split fees. It should be safe to remove this by now, but we need to be conscious that refunding old unified transactions in development environment won't work properly.

Reply:
@kewitz This line had no impact since we were forcing the `hostFeeInHostCurrency` to `0` at line 227.